packages/core/content-manager/admin/src/pages/App/tests/index.test.js

```diff
@@ -0,0 +1,474 @@
+/* eslint-disable no-irregular-whitespace */
+import React from 'react';
+import { createStore, combineReducers } from 'redux';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import { Provider } from 'react-redux';
+import { render, screen } from '@testing-library/react';
+import ContentManagerApp from '..';
+import cmReducers from '../../../reducers';
+import pluginId from '../../../pluginId';
+import useModels from '../useModels';
+
+jest.mock('../useModels', () =>
+  jest.fn(() => {
+    return {};
+  })
+);
+
+jest.mock('react-intl', () => ({
+  FormattedMessage: () => 'label',
+  useIntl: () => ({ formatMessage: jest.fn(() => 'label') }),
+}));
+jest.mock('@fortawesome/react-fontawesome', () => ({
+  FontAwesomeIcon: () => null,
+}));
+
+describe('Content manager | App | main', () => {
+  beforeEach(() => {
+    jest.resetModules(); // Most important - it clears the cache
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should not crash', () => {
```

Comment:
Can you add another case to make sure the App redirects to the first single type (homepage) when there is no collection type?